docs/getting-started.md

```diff
@@ -60,6 +60,9 @@ checked-in [deployment yaml](/build/yamls/antrea.yml):
 kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
 ```
 
+If you want to add Windows Nodes to your cluster, please refer to the installation instructions in
```

Comment:
forgot to wrap it in 80 chars?

Reply:
Sorry... forgot to update this file.

Done.